package.json

```diff
@@ -88,70 +88,68 @@
     "node": ">=8.16.0"
   },
   "devDependencies": {
-    "@types/node": "^11.13.12",
-    "@typescript-eslint/eslint-plugin": "^1.4.2",
-    "@typescript-eslint/parser": "^1.4.2",
+    "@types/node": "^12.0.10",
+    "@typescript-eslint/eslint-plugin": "^1.11.0",
+    "@typescript-eslint/parser": "^1.11.0",
     "JSONStream": "^1.3.5",
     "ajv-pack": "^0.3.1",
-    "autocannon": "^3.2.0",
+    "autocannon": "^4.0.0",
     "branch-comparer": "^0.4.0",
-    "concurrently": "^4.1.0",
+    "concurrently": "^4.1.1",
     "cors": "^2.8.5",
-    "coveralls": "^3.0.3",
+    "coveralls": "^3.0.4",
     "dns-prefetch-control": "^0.2.0",
     "eslint-import-resolver-node": "^0.3.2",
     "fast-json-body": "^1.1.0",
-    "fastify-plugin": "^1.5.0",
-    "fluent-schema": "^0.7.0",
-    "form-data": "^2.3.3",
-    "frameguard": "^3.0.0",
+    "fastify-plugin": "^1.6.0",
+    "fluent-schema": "^0.7.3",
+    "form-data": "^2.4.0",
+    "frameguard": "^3.1.0",
     "h2url": "^0.1.2",
-    "helmet": "^3.15.1",
-    "hide-powered-by": "^1.0.0",
-    "hsts": "^2.1.0",
-    "http-errors": "^1.7.1",
-    "ienoopen": "^1.0.0",
-    "joi": "^12.0.0",
+    "helmet": "^3.18.0",
+    "hide-powered-by": "^1.1.0",
+    "hsts": "^2.2.0",
+    "http-errors": "^1.7.3",
+    "ienoopen": "^1.1.0",
+    "@hapi/joi": "^15.1.0",
     "license-checker": "^25.0.1",
-    "lolex": "^4.0.1",
+    "lolex": "^4.1.0",
     "pre-commit": "^1.2.2",
     "proxyquire": "^2.1.0",
     "pump": "^3.0.0",
-    "semver": "^6.0.0",
-    "send": "^0.16.2",
-    "serve-static": "^1.13.2",
+    "semver": "^6.2.0",
+    "send": "^0.17.1",
+    "serve-static": "^1.14.1",
     "simple-get": "^3.0.3",
     "snazzy": "^8.0.0",
-    "split2": "^3.1.0",
+    "split2": "^3.1.1",
     "standard": "^12.0.1",
-    "tap": "^12.5.2",
-    "tap-mocha-reporter": "^3.0.7",
+    "tap": "^14.4.1",
+    "tap-mocha-reporter": "^4.0.1",
     "then-sleep": "^1.0.1",
-    "typescript": "^3.5.1",
-    "x-xss-protection": "^1.1.0"
+    "typescript": "^3.5.2",
+    "x-xss-protection": "^1.2.0"
   },
   "dependencies": {
     "abstract-logging": "^1.0.0",
-    "ajv": "^6.9.2",
-    "avvio": "^6.1.1",
-    "fast-json-stringify": "^1.15.0",
-    "find-my-way": "^2.0.0",
-    "flatstr": "^1.0.9",
-    "light-my-request": "^3.2.0",
+    "ajv": "^6.10.0",
+    "avvio": "^6.2.1",
+    "fast-json-stringify": "^1.15.3",
+    "find-my-way": "^2.1.0",
+    "flatstr": "^1.0.12",
+    "light-my-request": "^3.4.1",
     "middie": "^4.0.1",
-    "pino": "^5.11.1",
-    "proxy-addr": "^2.0.4",
-    "readable-stream": "^3.1.1",
-    "rfdc": "^1.1.2",
+    "pino": "^5.12.6",
+    "proxy-addr": "^2.0.5",
+    "readable-stream": "^3.4.0",
+    "rfdc": "^1.1.4",
     "secure-json-parse": "^1.0.0",
     "tiny-lru": "^6.0.1"
   },
   "greenkeeper": {
     "ignore": [
       "boom",
```

Comment:
We should remove `boom` from here as well.

Reply:
done